index.html

```diff
@@ -17,24 +17,24 @@
           <li class="nav__list-item"><a href="/" class="nav__link">contact</a></li>
         </ul>
       </nav>
-      <section class="header__partial--big">
-        <span class="header__decor--big">H</span>
+      <section class="header__partial--big header__partial--accent1">
```

Comment:
A gdzie tu podstawowe klasy? :) `header-partial header__partial--big header__partial--accent1`. Nadal się zastanawiam nad tym `accent1`, ale to wyjdzie w praniu z tymi kolorami.